packages/chat/src/components/NewChat/index.tsx

```diff
@@ -71,34 +71,35 @@ export const NewChat: React.FC<INewChat> = ({
   };
 
   // Trigger scroll when manually clicking the scroll button
-  const handleScrollToBottom = () => {
-    if (scrollableAreaRef.current) {
-      scrollableAreaRef.current.scrollTo({
-        top: scrollableAreaRef.current.scrollHeight,
-        behavior: 'smooth',
-      });
-    }
-  };
+  // const handleScrollToBottom = () => {
+  //   if (scrollableAreaRef.current) {
+  //     scrollableAreaRef.current.scrollTo({
+  //       top: scrollableAreaRef.current.scrollHeight,
+  //       behavior: 'smooth',
+  //     });
+  //   }
+  // };
```

Comment:
This will be refactored out in a future PR to a separate component - it shouldn't be here.